lhotse/recipes/ami.py

```diff
@@ -201,8 +201,12 @@ def download_audio(
             )
         elif mic == "mdm":
             for array in MDM_ARRAYS:
-                for channel in MDM_CHANNELS:
-                    wav_name = f"{item}.{array}-{channel}.wav"
+                if item == "IB4001" and array == "Array2":
```

Comment:
You could also just put a try/except block here and warn the user for failing downloads.

Reply:
good call! this would save a lot of effort.